keps/sig-node/127-user-namespaces/README.md

```diff
@@ -422,6 +422,34 @@ components that implement the interface.
 
 [kubeletVolumeHost-interface]: https://github.com/kubernetes/kubernetes/blob/36450ee422d57d53a3edaf960f86b356578fe996/pkg/volume/plugins.go#L322
 
+### Pod Security Standards (PSS) integration
+
+[Pod Security Standards](https://k8s.io/docs/concepts/security/pod-security-standards)
+define three different policies to broadly cover the whole security spectrum of
+Kubernetes, while the User Namespaces feature should integrate into them. This
+will happen only if the feature is graduated to GA, which _may_ result in
+changing the `Restricted` profile to disallow host user namespaces.
+
+With graduating the feature to beta, the Pod Security will relax in a controlled
+way for pods which enable user namespaces. This behavior can controlled by an API
+Server Feature Gate, which allows an early opt-in for end users. The overall
+burden to ensure that all nodes will honor user namespaces is on the cluster
+admin, though. The relaxation in detail means, that if user namespaces are
+enabled, then the following fields won't be restricted any more because they
+always have to refer to the user inside the container:
+
+- `spec.securityContext.runAsNonRoot`
+- `spec.containers[*].securityContext.runAsNonRoot`
+- `spec.initContainers[*].securityContext.runAsNonRoot`
+- `spec.ephemeralContainers[*].securityContext.runAsNonRoot`
+- `spec.securityContext.runAsUser`
+- `spec.containers[*].securityContext.runAsUser`
+- `spec.initContainers[*].securityContext.runAsUser`
+- `spec.ephemeralContainers[*].securityContext.runAsUser`
+- `spec.containers[*].securityContext.allowPrivilegeEscalation`
+- `spec.initContainers[*].securityContext.allowPrivilegeEscalation`
+- `spec.ephemeralContainers[*].securityContext.allowPrivilegeEscalation`
+
```

Comment:
what about capabilities?

Reply:
I feel like we could update the restricted profile to allow all capabilities. For instance, https://kubernetes.io/docs/concepts/workloads/pods/user-namespaces/ lists the behavior of `CAP_SYS_MODULE` and `CAP_SYS_ADMIN` inside a user namespace as only staying within the confines of the namespace. I'm fairly certain all capabilities are userns aware and could be given to the restricted profile without harm cc @giuseppe

Reply:
yes, we should be able to enable all capabilities in a user namespace.  The kernel will anyway handle them differently in a user namespace, e.g. it is not possible to really use `CAP_SYS_MODULE` from a user namespace

Reply:
Good, then we will remove the restrictions for capabilities as well :+1:

Reply:
Yes, all capabilities should be allowed :+1: